src/components/modules/nutrition/add-food/food-list-item-info.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+    <div v-if="foodNutrients">
+        <div class="food-info">
+            <div>
+                <span>Protein: </span>
+                <span>{{ foodNutrients.protein.value * portion | cutDigits}}{{ foodNutrients.protein.unit }}</span>
+            </div>
+            <div>
+                <span>Fat: </span>
+                <span>{{ foodNutrients.fat.value * portion | cutDigits }}{{ foodNutrients.fat.unit }}</span>
+            </div>
+            <div>
+                <span>Carbs: </span>
+                <span>{{ foodNutrients.carbs.value * portion | cutDigits }}{{ foodNutrients.carbs.unit }}</span>
+            </div>
+        </div>
+        <v-form v-if="showMore" class="portion-form">
+            <v-text-field
+                v-model="portion"
+                label="Number of portions"
+                type="number"
+            ></v-text-field>
+            <v-icon color="green" class="add-food-icon" @click="addFoodToCurrentDailyIntake">
+                add
+            </v-icon>
+        </v-form>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Watch } from 'vue-property-decorator';
+import { mapGetters } from 'vuex';
+
+import { cutDigits } from '@/filters/CutDigits.filter';
+
+import { FoodNutrient, Food, FoodInfo } from '@/models/nutrition';
+
+@Component({
+    filters: {
+        cutDigits
+    },
+    computed: {
+        ...mapGetters('nutritionModule', {getFoodNutrients: 'foodNutrients'})
+    }
+})
+export default class FoodListItemInfo extends Vue {
+    @Prop({default: null}) foodInfo!: FoodInfo;
```

Comment:
Usually it's more readable to add spaces after `{` and before `}`.

Reply:
Done